contribute/issues.md

```diff
@@ -0,0 +1,27 @@
+# List of labels for issues and PRs on OpenEBS Project
+
+This document provides list of labels and their purpose. This labels can be attached to issues and PRs.
+
+
+| Sr. No  |  Labels |  Label description |
+|---|---|---|
+| 1  | API  | Issue/PR related to API  |
+| 2 |  Architecting | If something needs design changes/brainstorming |
+| 3  | Bash  | Related to bash script |
+| 4  |  Bug |  If there’s an error, failure in workflow, fault etc.|
+| 5  | Documentation  |  Related to documentation |
+| 6  | Feature  | New / existing / required feature or enhancement |
+| 7  | Help(small)  | Need small help on docs (beginner level)|
+| 8  |  Help(medium) | Need help to fix code ( Intermediate level) |
+| 9  |  Help(large) | Need help on design changes and code review(Expert level) |
+| 10  | Improve | To improve some  |
+| 11  | Security  | Security related concerns  |
+| 12  | Testing  |  Needs/Adds tests |
+| 13  | WorkInProgress (Not WIP)  | Means work in progress |
+| 14  |  Website | Issues related to website  |
+| 15  | area/maya  | Maya Specific  |
```

Comment:
what is the difference between a label (without area) and with area. it will be good to have the reasoning about the classification of labels in the beginning of this doc.